app/models/case_log.rb

```diff
@@ -619,9 +619,9 @@ def get_has_benefits
   end
 
   def get_lettype
-    return unless renttype.present? && needstype.present? && owning_organisation.present? && owning_organisation[:provider_type].present?
+    return unless rent_type.present? && needstype.present? && owning_organisation.present? && owning_organisation[:provider_type].present?
 
-    case RENT_TYPE_MAPPING_LABELS[renttype]
+    case RENT_TYPE_MAPPING_LABELS[RENT_TYPE_MAPPING[rent_type]]
```

Comment:
Since we are refactoring this, any reason why we use the labels for comparison?

Reply:
Just for readability, so you know what each case actually represents